### ISSUE TITLE
Op type not registered 'NonMaxSuppressionV3' deploying model in Cloud ML

### ISSUE THREAD

**Issue by samclearman** [tensorflow/models#5210]
New issue for https://github.com/tensorflow/models/issues/5171

**Comment by samclearman** [comment]
@karmel I'm creating a new issue here because I can't reopen the original.  As the OP noted, the model does work with TF 1.9

**Comment by karmel** [comment]
@samclearman -- it looks like the linked issue was resolved. Can you recap here what your issue is, what you have tried, and what the errors are? It's not clear to me from the linked thread.

**Comment by samclearman** [comment]
The issue is that when I try to upload my trained model to inference engine, it fails with the message "Op type not registered 'NonMaxSuppressionV3'".

The command that fails would be something like "gcloud ml-engine versions create v1 --model "aq_frcnn_v1" --origin $DEPLOYMENT_SOURCE 
--runtime-version=1.9" (from the original thread).

It seems that the problem is OS X specific?

**Comment by SunnQinnWenn** [comment]
Having the same error message too: 
`ERROR: (gcloud.ml-engine.versions.create) Bad model detected with error:  "Failed to load model: Loading servable: {name: default version: 1} failed: Not found: Op type not registered 'NonMaxSuppressionV3' in binary running on localhost. Make sure the Op and Kernel are registered in the binary running in this process.\n\n (Error code: 0)"`

**Comment by adishupadhyay** [comment]
> Having the same error message too:
> `ERROR: (gcloud.ml-engine.versions.create) Bad model detected with error: "Failed to load model: Loading servable: {name: default version: 1} failed: Not found: Op type not registered 'NonMaxSuppressionV3' in binary running on localhost. Make sure the Op and Kernel are registered in the binary running in this process.\n\n (Error code: 0)"`

Hi Any Update on below issue.i am facing same issue
Op type not registered 'NonMaxSuppressionV5' in binary running on 9666156f8e0e. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed

**Comment by kumariko** [comment]
@samclearman We are checking to see if you still need help on this issue? Could you please have a look on the [link](https://github.com/GoogleCloudPlatform/cloudml-samples/issues/224) and let us know if it helps? Thanks!
